db/scripts/cleanup_scheduled_calls.js

```diff
@@ -5,7 +5,10 @@ async function getDb() {
   dotenv.config();
 
   const pgp = require("pg-promise")({});
-  const connectionURL = process.env.DATABASE_URL;
+  const connectionURL =
+    process.env.NODE_ENV === "test"
+      ? process.env.TEST_DATABASE_URL
+      : process.env.DATABASE_URL;
   return pgp({
     connectionString: connectionURL,
     ssl: { rejectUnauthorized: false },
```

Comment:
The test database on circleCI (and local dev) does not use an SSL connection, so we will need to update this to only set in production (like https://github.com/madetech/nhs-virtual-visit/blob/master/src/gateways/Database/index.js#L17-L19

Reply:
Done